tests/components/color_picker.test.ts

```diff
@@ -51,12 +51,15 @@ describe("Color Picker buttons", () => {
     expect(onColorPicked).toHaveBeenCalledWith("");
   });
 
-  test("Click on '+' button toggle the custom color part", async () => {
+  test("Clicking on '+', custom section except custom added colors must toggle the custom color picker", async () => {
     await mountColorPicker();
-    await simulateClick(".o-color-picker-toggler");
-    expect(fixture.querySelector(".o-custom-selector")).toBeDefined();
-    await simulateClick(".o-color-picker-toggler");
-    expect(fixture.querySelector(".o-custom-selector")).toBeNull();
+    const elements = fixture.querySelectorAll(".o-color-picker-toggler");
+    for (const element of elements) {
+      await simulateClick(element);
```

Comment:
See my previous comment :) 
```suggestion
    const selectors = [".o-color-picker-section-name", ".colors-grid", ".o-color-picker-plus"]
    for (const selector of selectors) {
      const element = fixture.querySelector(selector)
      await simulateClick(element);
```

Reply:
Yes I thought so. But the classes .o-color-picker-section-name and .colors-grid are applied to Standard as well as Custom section. Then to explicitly access custom section name and grid, how about we add classes like o-custom-section-name & o-custom-colors-grid?

Reply:
Oh yes, you're right...
let's keep it like that then and add `.o-color-picker-toggler` on the "+" as well so it's tested. That's good enough :)